content/en/docs/components/pipelines/caching-v2.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Caching v2"
+description = "Getting started with Kubeflow Pipelines caching v2"
+weight = 50
+
++++
+{{% beta-status
+feedbacklink="https://github.com/kubeflow/pipelines/issues" %}}
+
+Starting from [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/sdk/v2/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports step caching capabilities in both standalone deployment and GCP hosted deployment.
+
+## Before you start
+
+This guide tells you the basic concepts of Kubeflow Pipelines caching and how to use it. 
+This guide assumes that you already have Kubeflow Pipelines installed or want to use standalone or GCP hosted deployment options in the [Kubeflow Pipelines deployment 
+guide](/docs/components/pipelines/installation/) to deploy Kubeflow Pipelines.
+
+## What is step caching?
+
+Kubeflow Pipelines caching provides step-level output caching. 
+And caching is enabled by default for all tasks of pipelines built with [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/sdk/v2/) using `kfp.dsl.PipelineExecutionMode.V2_COMPATIBLE` mode.
+When Kubeflow Pipeline runs a pipeline, it checks to see whether 
+an execution exists in Kubeflow Pipeline with the interface of each pipeline task.
+The task's interface is defined as the combination of the pipeline task spec(base image, command, args), the pipeline task's inputs(artifacts and parameters), and the pipeline task's outputs(artifacts and parameters). 
```

Comment:
NIT: For artifact, do we compare the artifact ID, or the fingerprint of artifact content, or etc.? It will be great to share the mechanism so user understands better about when the cache will be hit.

Reply:
These are currently implementation details in v2 compatible mode. I'd prefer deferring sharing the mechanism until we expose KFP IR.

Reply:
I see, what I meant is to help user understand what makes one execution to hit cache: It is easy to understand `parameter`: For example, if two string values equal to each other, there is a match. But what about `artifact`? How to determine if two artifacts equal to each other?

Reply:
the id of artifact is the same

Reply:
Great! Feel free to add this info in this PR, or defer it to later when KFP IR is ready, like you mentioned.

Reply:
Do we expose MLMD model to users? For example, when we explain the id or name of the artifact, will it be kind of confusing without exposing KFP IR?

Reply:
These concepts are exposed in KFP UI, we show input artifacts and their name and id.

But for ID, we should further explain the easily understandable implication -- if the artifact producer task is not cached, then this will be a new artifact with different ID, and this task won't be cached.

I think explaining like this will make people understand without much knowledge of mlmd, how do you think?

Reply:
done